packages/EasyPagination/src/Paginators/DoctrineDbalLengthAwarePaginator.php

```diff
@@ -64,7 +64,16 @@ protected function doGetResult($queryBuilder): array
      */
     protected function doGetTotalItems($queryBuilder, string $countAlias): int
     {
-        $result = (array)$this->conn->fetchAssoc($queryBuilder->getSQL(), $queryBuilder->getParameters());
+        // @TODO remove this method_exists check when dbal will be updated to version with fetchAssociative method (>=2.11)
```

Comment:
Pls update a little to `@TODO remove this method_exists check when doctrine/dbal is updated to a version containing the fetchAssociative method (>=2.11)`

Reply:
Done